honeybee_vtk/write.py

```diff
@@ -102,21 +110,22 @@ def write_vtk(file_path, *, file_name=None, target_folder=None, include_grids=Tr
 
     # remove extension if provided by user
     file_name = file_name if not file_name.lower().endswith('.zip') else file_name[:-4]
-
-    target_folder = target_folder or os.getcwd()
-    zip_file = os.path.join(target_folder, file_name + '.zip')
```

Comment:
why are you removing the check for None target folder?

Reply:
I did not remove it. Just shifted it [here](https://github.com/ladybug-tools/honeybee-vtk/pull/9/files#diff-74d6244911d9a54aabb679036a0eb51ae1bc14166f91efe71c6e55e2415b942bR71)